azavea/tilegarden
# Feature/Demo webpage

## Overview
This PR does the following things:
* New demo material has been created to show off the functionality of the project. A `gh-pages` branch has been created to host these demos, which connect to a persistent deployment of the project. `gh-pages` was branched from a few commits prior to this PR in order to make sure automated deployment works whenever this PR gets merged.
* `.travis.yml` has been updated to automatically deploy the project to both AWS and `gh-pages` on successful builds of `master`. Build artifacts have been un-gitignored in `gh-pages` in order to facilitate this.
* Not strictly part of this PR, but the RDS instance has been updated so that production lambda functions can have access to the street/parcel data sets.


### Demo
![image](https://user-images.githubusercontent.com/16990679/43211812-6d44a538-9000-11e8-80f7-260791e507fe.png)
![image](https://user-images.githubusercontent.com/16990679/43211844-80eb6176-9000-11e8-8be2-b7a2c7f13307.png)
![image](https://user-images.githubusercontent.com/16990679/43211870-95df26c6-9000-11e8-879f-05a3355f8497.png)
![image](https://user-images.githubusercontent.com/16990679/43211888-a341262a-9000-11e8-939c-8345efe72751.png)
![image](https://user-images.githubusercontent.com/16990679/43211905-afd91e38-9000-11e8-924b-4cb7e3c9b624.png)



### Notes
 * I've tested the deployment part of this PR on a dummy branch so everything should work when this gets merged. 
 * Graphic design of the demos is still WIP, and I need to get feedback from #fellowship-program once the GitHub pages is live.


## Testing Instructions

 * Run the local server with `./scripts/update`, `./scripts/server`.
 * Open up `index.html` and navigate around. All the demos should work properly.


Resolves #36, resolves #47,  resolves #71, resolves #62,
